client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -30,6 +32,17 @@ interface Props {
 
 export default function Login({ handleSubmit }: Props): JSX.Element {
   const classes = useStyles();
+  const { isOpen, setIsOpen, currentStep, setCurrentStep } = useTour();
```

Comment:
Everything in the login form is self explanatory, I don't we need a tour bubble here :)

Reply:
OK. Done.